rosetta/docs/GPU_performance.md

```diff
@@ -0,0 +1,129 @@
+# Tips for High-Performance LLMs with JAX and XLA 
```

Comment:
Shall we add a link from README.md to this page?

Reply:
That's a good idea. Maybe we spend a couple of mins in PAXML standup meeting tomorrow to discuss that as well.

Reply:
Added suggestion to PR description. Thanks.